src/api/routes/index.js

```diff
@@ -0,0 +1,21 @@
+import { Router } from 'express';
+import photosController from 'controllers';
+import { BAD_REQUEST } from 'constants/error.types';
+
+const photosRouter = Router();
+
+photosRouter.get('/', async (req, res) => {
+  const { query: { id }} = req;
+
+  try {
+    const list = await photosController(id);
```

Comment:
Could you explain your reasoning behind structuring your code this way? Why separate just this code?

Reply:
Have you considered the use of middleware for validation?

Reply:
I did consider using a middleware for validation.
In the moment, my thought was "If its just a single endpoint and the validation needs to be done only for that endpoint, I'll do it in the controller".

If the validation was required across multiple endpoints, I would've tried to separate it from a controller and would've ended up using a middleware.